pages/user/order/countDown.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { useState, useEffect } from 'react';
+
+interface Props {
+  format: boolean;
+  className?: string;
+}
+
+const getDifference = (d1: Date, d2: Date) => {
+  const hours = (d2.getHours() - d1.getHours()) % 24;
+  const minutes = (d2.getMinutes() - d1.getMinutes()) % 60;
+  const seconds = (d2.getSeconds() - d1.getSeconds()) % 60;
+  const res: Date = new Date();
+  res.setHours(hours);
+  res.setMinutes(minutes);
+  res.setSeconds(seconds);
+  return res;
+};
+
+function padZero(n: string, width: number) {
```

Comment:
I think addZeroPadding should be better name for this function, since padZero is not a verb.

Reply:
But “pad” is a verb: https://www.merriam-webster.com/dictionary/pad 🤔

Reply:
I see